src/routes/authRouter.ts

```diff
@@ -1,14 +1,15 @@
-import {validation,isUserExist, isAccountVerified} from "../middlewares/validation";
-import authControllers from "../modules/auth/controller/authControllers";
 import { Router } from "express";
-import { emailSchema, registerSchema} from "../modules/auth/validation/authValidations";
+import authControllers from "../modules/auth/controller/authControllers";
+import { validation, isUserExist, isAccountVerified, checkLoginUser } from "../middlewares/validation";
+import { emailSchema, registerSchema, loginSchema } from "../modules/auth/validation/authValidations";
 
 
 const router: Router = Router();
 
 router.post("/register", validation(registerSchema), isUserExist, authControllers.registerUser);
 router.get("/verify-email/:token", isAccountVerified, authControllers.verifyEmail);
 router.post("/send-verify-email", validation(emailSchema), isAccountVerified, authControllers.sendVerifyEmail);
+router.post("/login", validation(loginSchema), checkLoginUser, authControllers.loginUser);

```

Comment:
instead of using loginSchema, just use registerSchema as it is working the same. And remember to remove loginSchema into codebase.